server/v1/middlewares/Auth.js

```diff
@@ -0,0 +1,11 @@
+const Auth = {
+    right(req, res, next) {
+        req.userData = 1;
```

Comment:
Why do you hardcode 1 here? If you are authenticating users, its better to check if the `user id` in the `userData` is in your db. It is far more efficient than hardcoding `1`. Hardcoding 1 means its only user 1 that can fetch the data. Please rectify this

Reply:
I have removed that part of the code. I will rewrite it to do a proper admin rights under **users**